frontend/.env.example

```
@@ -0,0 +1,21 @@
+ENABLE_ADMIN_API=
+
+PAYMENTS_ENABLED=
+NEXT_PUBLIC_PAYMENTS_ENABLED=
+STRIPE_SECRET_KEY=
+STRIPE_WEBHOOK_SECRET=
+
+STRIPE_MODE=
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=
+
+NEXT_PUBLIC_SITE_URL=
+NEXT_PUBLIC_SITE_URL=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_SITE_URL key is duplicated

(DuplicatedKey)